kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver_with_constraints.h

```diff
@@ -0,0 +1,630 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Aditya Ghantasala / Navaneeth K Narayanan
+//
+//
+
+#ifndef KRATOS_SOLVING_STRATEGIES_BUILDER_AND_SOLVERS_RESIDUALBASED_BLOCK_BUILDER_AND_SOLVER_WITH_CONSTRAINTS_H_
+#define KRATOS_SOLVING_STRATEGIES_BUILDER_AND_SOLVERS_RESIDUALBASED_BLOCK_BUILDER_AND_SOLVER_WITH_CONSTRAINTS_H_
+
+/* System includes */
+
+#include "utilities/openmp_utils.h"
+#include <unordered_set>
+#include <algorithm>
+/* External includes */
+#include "boost/smart_ptr.hpp"
+
+#include "utilities/timer.h"
+#include "utilities/constraint.h"
+#include "utilities/multipoint_constraint.h"
```

Comment:
Remove multipoint_constraint.h.

Reply:
This is a leftover line code during clean up I will remove it. As we can see there are no specifics of multipoint_constraint.h used in the builder and solver. Also The "constraint" is meant to be an interface.

Reply:
ok